client/src/pages/About/Overview/DruggableGenome.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from '@mui/material';
+import { NoteBox } from 'components/About/NoteBox/NoteBox';
+
+import './DruggableGenome.scss';
+
+export const AboutDruggableGenome: React.FC = () => {
+    return (
+        <>
+        <h1>The Druggable Genome</h1>
+        <h3>What is druggability?</h3>
+        <p>Druggability refers to the potential of a protein to interact with drugs or other chemical compounds based on its structural characteristics. As originally described by Russ Lampel in 2006, a protein is considered druggable if its structure supports interactions with small molecules or other therapeutic agents. Proteins lacking structural features or a favorable location for physical interaction, or those that cannot be modified to facilitate such interactions, are not deemed druggable.
```

Comment:
I believe this should be "Russ et al" or "Russ and Lampel" its two different last names, not one person's name.

Reply:
changed to Russ and Lampel